src/EnergyPlus/api/datatransfer.cc

```diff
@@ -259,6 +259,19 @@ int getActuatorHandle(const char* componentType, const char* controlType, const
         std::string const actuatorIDUC = EnergyPlus::UtilityRoutines::MakeUPPERCase(availActuator.UniqueIDName);
         std::string const actuatorControlUC = EnergyPlus::UtilityRoutines::MakeUPPERCase(availActuator.ControlTypeName);
         if (typeUC == actuatorTypeUC && keyUC == actuatorIDUC && controlUC == actuatorControlUC) {
+
+            // If the handle is already used by an IDF EnergyManagementSystem:Actuator, we should warn the user
+            for (int ActuatorLoopUsed = 1; ActuatorLoopUsed <= EnergyPlus::DataRuntimeLanguage::numActuatorsUsed; ++ActuatorLoopUsed) {
+                auto const & usedActuator = EnergyPlus::DataRuntimeLanguage::EMSActuatorUsed(ActuatorLoopUsed);
+                if (usedActuator.ActuatorVariableNum == handle) {
+                    EnergyPlus::ShowWarningError("Data Exchange API: An EnergyManagementSystem:Actuator seems to be already defined in the EnergyPlus File and named " + usedActuator.Name);
+                    EnergyPlus::ShowContinueError("Occurred for componentType='" + typeUC + "', controlType='" + controlUC
+                            + "', uniqueKey='" + keyUC + "'.");
+                    EnergyPlus::ShowContinueError("The getActuatorHandle function will still return the handle (= " + std::to_string(handle)
+                            + ") but caller should take note that there is a risk of overwritting.");
+                }
+            }
+
```

Comment:
Should this be wrapped into a 

```
if (!EnergyPlus::DataGlobals::eplusRunningViaAPI)
```

?

Reply:
That would override the EMS actuator. It seems only one or the other should be used? Or figure out which should have priority.

Reply:
Oh, this does choose Python as the overriding action. I guess then .. would this even be called if !EnergyPlus::DataGlobals::eplusRunningViaAPI were true?

Reply:
I haven't tried the API, just the python plugin and it's definitely called.

I need to add a graph showing the defect file for 8084 before and after... in the defect file the EMS actuator is defined but not actually used by an EMS program. But because it is, the python stuff  isn't working correctly. I'll add that tomorrow.